src/action.ts

```diff
@@ -39,7 +39,10 @@ import {
 async function action(pr: PullRequest): Promise<void> {
   const tfInstance = getInput('api_url');
 
-  const api = new TestingFarmAPI(tfInstance);
+  const api = new TestingFarmAPI(
```

Comment:
Only nit-pick. I would prefer to add there a link to testing-farm repo I would like to find usage. But not blocking this PR. WDYT.

Reply:
Good point, done